### ISSUE TITLE
The first cov2d layer

### ISSUE THREAD

**Issue by melhzy** [Bingmang/pytorch-cifar10-notebook#1]
Hi Todd,

You have done a great job in your neural networks training. However, I have a question for you. It seems you have modified/tuned/altered ResNet's layers and the associated values.

For example, in the beginning of ResNet50:

> ResNet PyTorch Implementation: Conv2d(3, 64, kernel_size=(7, 7), stride=(2, 2), padding=(3, 3), bias=False)
> Yours:    Conv2d(3, 64, kernel_size=(3, 3), stride=(1, 1), padding=(1, 1), bias=False)

In the end of ResNet50:

> ResNet PyTorch Implementation: AdaptiveAvgPool2d(output_size=(1, 1))
> Yours:    AvgPool2d(kernel_size=4, stride=4, padding=0)

I also see different configurations in other layers.

How did u come up with these changes? Would you mind sharing what have you done? Thanks!

Sincerely,

Z.
